### PR TITLE
Update working_with_projections.rst

### PR DESCRIPTION
**Pull request by rmarzocchi84**
Add NTv2 specification. 
(https://gis.stackexchange.com/questions/298465/integrating-ntv2-transformation-in-qgis)

### Description

Goal:

Ticket: #

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by rmarzocchi84** [comment]
nadgrids_example

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 1]
```diff
@@ -222,6 +222,17 @@ only two parameters to define a user CRS:
 To create a new CRS, click the |signPlus| :sup:`Add new CRS` button and
 enter a descriptive name and the CRS parameters.
 
+Integrate a NTv2-transformation in QGIS
```
Comment (havatv) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]:
```suggestion
Integrate an NTv2-transformation in QGIS 
```

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 2]
```diff
@@ -222,6 +222,17 @@ only two parameters to define a user CRS:
 To create a new CRS, click the |signPlus| :sup:`Add new CRS` button and
 enter a descriptive name and the CRS parameters.
 
+Integrate a NTv2-transformation in QGIS 
+""""""""""""""""""""""""""""""""""""""""""""""
+To integrate a NTv2 transformation file in QGIS you need one more step:
```
Comment (havatv) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]:
```suggestion
To integrate an NTv2 transformation file in QGIS you need one more step: 
```

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 3]
```diff
@@ -222,6 +222,17 @@ only two parameters to define a user CRS:
 To create a new CRS, click the |signPlus| :sup:`Add new CRS` button and
 enter a descriptive name and the CRS parameters.
 
+Integrate a NTv2-transformation in QGIS 
+""""""""""""""""""""""""""""""""""""""""""""""
+To integrate a NTv2 transformation file in QGIS you need one more step: 
+
+1. Add the NTv2 file (.gsb) in the CRS folder of QGIS (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)
```
Comment (havatv) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]:
```suggestion
1. Place the NTv2 file (.gsb) in the CRS/Proj folder that QGIS uses (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)
```

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 4]
```diff
@@ -222,6 +222,17 @@ only two parameters to define a user CRS:
...
+2. Add in the proj.4 definition the parameter **nadgrids** (+nadgrids=namefile.gsb)
```
Comment (havatv) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]:
```suggestion
2. Add **nadgrids** (+nadgrids=nameofthefile.gsb) to the Proj4 definition in the :guilabel:`Parameters` field of the :guilabel:`Custom Coordinate Reference System Dialog` (:menuselection:`Settings --> Custom Projections...`).
```

**Comment by rmarzocchi84** [comment]
Thanks for your suggestion. It is the first time I try to contribute actively to the manual of QGIS and obviously I am not expert about the better way to do it

**Comment by rmarzocchi84** [comment]
@havatv sorry for the stupid question, but I need to close the pull request or only you as reviewer will close the pull request when you update the qgis official documentation?

**Comment by havatv** [comment]
@rmarzocchi84 - First of all, thank you so much for contributing to improving the QGIS documentation, and I hope that this will not be your last contribution!
if you close the pull request, you indicate that you do not want it to be merged - so don't do that.
Those that have the rights can merge your pull request any time.  I suggest that we wait for input from @DelazJ before we get this pull request merged.

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 5]
```diff
@@ -222,6 +222,19 @@ only two parameters to define a user CRS:
 To create a new CRS, click the |signPlus| :sup:`Add new CRS` button and
 enter a descriptive name and the CRS parameters.
 
+Integrate an NTv2-transformation in QGIS 
+""""""""""""""""""""""""""""""""""""""""""""""
+To integrate an NTv2 transformation file in QGIS you need one more step: 
+
+1. Place the NTv2 file (.gsb) in the CRS/Proj folder that QGIS uses (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/working_with_projections/working_with_projections.rst]: On Ubuntu (18.04, but also earlier versions) the location is /usr/share/proj/

[the remark this answers is not in the document]
Comment (rmarzocchi84) [reply source/docs/user_manual/working_with_projections/working_with_projections.rst]: I confirm the Ubuntu/debian location

**Comment by rmarzocchi84** [comment]
> Looking at the screenshot, I'm a bit lost in the information I should pick there at first glance. I think the long list of generated CRSes, with no highlight on selected one, makes the reading less easy (though I finally found that the one you add was at the end of the list).
> Maybe we can use a less "crowded dialog", ie:
> 
> * menu Settings --> User profiles --> New Profile to open a [new clean instance](https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles) of QGIS
> * Add the UTM29 projection as shown in the previous screenshot (optional?)
> * and add your 3003_grigliati_vicenza projection afterwards
> * use the screenshots with these two projections
> * delete the user profile afterwards (you'l have to delete it from the file browser - no direct tool from QGIS, yet!)

ntv2_pic

**Comment by DelazJ** [comment]
@rmarzocchi84 Yes thanks. It'd be super OK if we had it in English and the height of the definition frame is reduced (should be wider than higher). Also let's keep expanded the "prova" section as you did in the previous one.

**Comment by rmarzocchi84** [comment]
> @rmarzocchi84 Yes thanks. It'd be super OK if we had it in English and the height of the definition frame is reduced (should be wider than higher). Also let's keep expanded the "prova" section as you did in the previous one.


ntv2_pic

@DelazJ

**Comment by DelazJ** [comment]
Looks like you miss the 3003_grids_vicenza crs in the list now... Add it next to the UTM 29 and it should be OK. Thanks.

**Comment by DelazJ** [comment]
May I? @rmarzocchi84 , looks like you forgot this.😁

**Comment by rmarzocchi84** [comment]
grigliati
sorry. you are right!

**Pull request merged**